src/chapters/sql-part1/queries.rst

```diff
@@ -0,0 +1,256 @@
+
+In the case of inviting people the Johnson's vow renewal, we may want to invite Eliza's sister, Felicity, as well. However, we do not have confirmed Felicity's dietary preferences or whether or not she is coming.
+
+.. sourcecode:: mysql
+
+   INSERT INTO JohnsonVowRenewal (GuestID, LastName, FirstName)
+   VALUES (186, "Johnson", "Felicity");
+
+By adding the column names in parantheses after the table name, we have specified that we are adding a new row of data to the table, but we only have values for the columns: ``GuestID``, ``LastName``, and ``FirstName``.
+
+.. admonition:: Note
+
+   When we use this method, any column that doesn't have a specified value for the new row will have a ``null`` value. 
+
+Adding a column
+^^^^^^^^^^^^^^^
+
+Sometimes, we may also need to add a column to our table. Some of the caterers Mary works with have asked that she confirm how many people are of drinking age.
+We may now want to add a column to our ``LiWedding`` table that contains either ``1`` or ``0`` depending on whether the guest is legally old enough to drink.
+
+To add a column, we need to start with an ``ALTER TABLE`` statement. ``ALTER TABLE`` can be used to perform different operations, so in our case, we will also need to specify that we want to ``ADD`` a column.
+
+.. sourcecode:: mysql
+
+   ALTER TABLE LiWedding
+   ADD CanDrink boolean;
+
+This adds the ``CanDrink`` column to the ``LiWedding`` table, but it does *not* fill that column with values. We will need to update each guest's entry in the table once we confirm how old they are.
+
+.. admonition:: Note
+
+   For more on the ``ALTER TABLE`` statement and how many different ways it can be used, check out `w3schools <https://www.w3schools.com/sql/sql_alter.asp>`_.
+
+Read
+----
+
+When reading data, we don't want to modify anything, we just want to know what is there. 
+In order to get information from a table, we need to use a ``SELECT`` statement. 
+
+``SELECT`` statements have a few different components to them. We need to know what we are selecting, which table the information is in, and if necessary, we can also use ``WHERE`` to apply a conditional.
+In general, ``SELECT`` statements look like the following:
+
+.. sourcecode:: mysql
+
+   SELECT ColumnName1, ColumnName2, ...
+   FROM TableName
+   WHERE some conditional is true.
+
+If Mary wants to get the information of all of the guests who are vegetarian at the Li wedding, we need to use a ``SELECT`` statement to pull the first and last names of guests who will be in attendance and are vegetarian.
+So, we will ``SELECT`` the ``LastName`` and ``FirstName`` columns ``FROM`` the ``LiWedding`` table ``WHERE`` the value of ``Attending`` is ``TRUE`` and the value of ``Diet`` is ``"vegetarian"``.
+
+.. sourcecode:: mysql
+   :linenos:
+
+   SELECT LastName, FirstName
+   FROM LiWedding
+   WHERE (Attending == 0) AND (Diet == "vegetarian");
```

Comment:
``Attending == 1`` to check for ``true``.

Reply:
Done! Thanks!